app/Jobs/PruneAccessFromAccessInactiveUsers.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Jobs;
+
+use App\Models\User;
+use Carbon\CarbonImmutable;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Foundation\Bus\Dispatchable;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Queue\SerializesModels;
+
+class PruneAccessFromAccessInactiveUsers implements ShouldQueue
+{
+    use Dispatchable;
+    use InteractsWithQueue;
+    use Queueable;
+    use SerializesModels;
+
+    /**
+     * Create a new job instance.
+     */
+    public function __construct()
+    {
+        $this->queue = 'jedi';
+    }
+
+    /**
+     * Execute the job.
+     */
+    public function handle(): void
+    {
+        User::accessInactive(CarbonImmutable::now()->subDay())
```

Comment:
The timestamp passed in here is 1 day ago, so the DB results will contain users that were inactive as of 1 day ago

Reply:
This is the line I was missing. Thanks.